app/pages/ReassessmentPage.scala

```diff
@@ -25,15 +25,15 @@ import scala.util.{Failure, Success, Try}
 case class ReassessmentPage(
   recordId: String,
   index: Int
-) extends QuestionPage[AssessmentAnswer] {
+) extends QuestionPage[ReassessmentAnswer] {
   override def path: JsPath = JsPath \ "reassessments" \ recordId \ index
 
   override def cleanup(
-    value: Option[AssessmentAnswer],
+    value: Option[ReassessmentAnswer],
     updatedUserAnswers: UserAnswers,
     originalUserAnswers: UserAnswers
   ): Try[UserAnswers] =
-    if (value.contains(AssessmentAnswer.NoExemption)) {
+    if (value.exists(x => x.answer == AssessmentAnswer.NoExemption)) {
```

Comment:
try not to use x as variable - I recall liandra saying something about this not being allowed in the final codebase

Reply:
oh.. renamed it it now